webrender/src/resource_cache.rs

```diff
@@ -651,6 +674,87 @@ impl ResourceCache {
             }
         }
     }
+    fn update_texture_cache(cached_images: &mut ResourceClassCache<ImageRequest, CachedImageInfo>,
```

Comment:
This function is extracted from:
https://github.com/servo/webrender/pull/997/commits/b4e91e1b7832ef9a983a5243042d8dd0dff9c91c#diff-77cbdf7ba9ebae81feb38a64c21b8454L669